### ISSUE TITLE
Integrate sandbox system in steam

### ISSUE THREAD

**Issue by ekaris** [ValveSoftware/steam-for-linux#4218]
It is a good idea to have sandbox implemented on steam client

**Comment by vincent-t** [comment]
@ekaris I think you mean the chromium sandbox?

**Comment by ekaris** [comment]
No, that's the browser sandbox, what I was referring to is running steam in a sandbox and all its games in a sandbox for example like with firejail - it has upstream support for steam.

**Comment by ekaris** [comment]
"Other vulnerabilities that are favorite attack vectors? The personal devices employees bring into the office on which they’ve allowed their kids to load Steam games, and which the workers then connect to the network."

Rob Joyce (TAO, NSA) just said Steam games are a security threat #enigma2016

http://www.wired.com/2016/01/nsa-hacker-chief-explains-how-to-keep-him-out-of-your-system/

**Comment by ekaris** [comment]
Even Itch.io app now already has sandbox

**Comment by ghost** [comment]
There are multiple solutions in my opinion:
1. A verified flatpak
2. Upgrading the current CEF sandbox similar to base chromium (might require upstream work):
<img width="402" height="275" alt="Image" src="https://github.com/user-attachments/assets/86d242d7-bf58-41fa-8858-4ad6f3593536" />

Personally I prefer solution 1. since it is fastest to implement since CEF development is so slow they don't even support Wayland natively yet.
